modules/builds-source-secrets-entitlements.adoc

```diff
@@ -12,6 +12,16 @@ Builds that use Red Hat subscriptions to install content must include the entitl
 
 You must have access to Red Hat entitlements through your subscription, and the entitlements must have separate public and private key files.
 
+
+[TIP]
+====
+When performing an Entitlement Build using {op-system-base-full} 7 be sure to have the following in your Dockerfile before any `yum` commands:
```

Comment:
A small tweak for second person perspective (preferred way per ISG), and to provide an [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) for the Dockerfile command:
```suggestion
When you perform an Entitlement Build using {op-system-base-full} 7, you must have the following command in your Dockerfile before you run any `yum` commands:
```
The second suggestion about object references is from the supplemental guidelines, and it seems to just mean when you have a literal value within the sentence itself 🙂 but even in an indirect case like this one, I think it adds some clarity